contracts/standard/arbitration/composed-arbitrable/example/ArbitrableTokenList.sol

```diff
@@ -0,0 +1,594 @@
+        Submitted, // The item has been submitted.
+        ClearingRequested, // The item is registered, but someone has requested to remove it.
+        PreventiveClearingRequested // The item has never been registered, but someone asked to clear it preemptively to avoid it being shown as not registered during the dispute resolution process.
+    }
+
+    /* Structs */
+
+    struct Item {
+        ItemStatus status; // Status of the item.
+        uint lastAction; // Time of the last action.
+        // The amount of funds placed at stake for this item. Does not include arbitrationFees
+        uint balance;
+        uint challengeReward; // The challengeReward for the item, if there a party made a request.
+    }
+
+    /* Events */
+
+    /**
+     *  @dev Called when the item's status changes or when it is contested/resolved.
+     *  @param submitter Address of the submitter, if any.
+     *  @param challenger Address of the challenger, if any.
+     *  @param tokenID The tokenID of the item.
+     *  @param status The status of the item.
+     *  @param disputed The item is being disputed.
+     */
+    event ItemStatusChange(
+        address indexed submitter,
+        address indexed challenger,
+        bytes32 indexed tokenID,
+        ItemStatus status,
+        bool disputed
+    );
+
+    /* Storage */
+
+    // Settings
+    uint public challengeReward; // The challengeReward to put to submit/clear/challenge an item in addition of arbitration fees.
+    uint public timeToChallenge; // The time before which an action is executable if not challenged.
+    address public challengeRewardGovernor; // Is allowed to update challengeReward and challengeRewardGovernor
+
+    // Ruling Options
+    uint8 constant REGISTER = 1;
+    uint8 constant CLEAR = 2;
+
+    // Items
+    mapping(bytes32 => Item) public items;
+    mapping(uint => bytes32) public disputeIDToItem;
+    bytes32[] public itemsList;
+
+    // Agreement and Item extension
+    mapping(bytes32 => uint) public itemIDToAgreementCount;
+    mapping(bytes32 => bytes32) public agreementIDtoItemID;
+
+    /* Constructor */
+
+    /**
+     *  @dev Constructs the arbitrable permission list and sets the type.
+     *  @param _arbitrator The chosen arbitrator.
+     *  @param _arbitratorExtraData Extra data for the arbitrator contract.
+     *  @param _metaEvidence The URL of the meta evidence object.
+     *  @param _challengeReward The amount in Weis of deposit required for a submission or a challenge in addition of the arbitration fees.
+     *  @param _challengeRewardGovernor The challengeReward governor
+     *  @param _timeToChallenge The time in seconds, other parties have to challenge.
+     *  @param _feeGovernor The fee governor of this contract.
+     *  @param _stake The stake parameter for sharing fees.
+     */
+    constructor(
+        Arbitrator _arbitrator,
```

Comment:
I've changed the order of parameters.
Regarding the arbitrator storage, I imagine there is no need to assign it again here since it is passed to parent constructors and stored inside Arbitrable.sol.

Reply:
Yeah